api-reference/50 Common/utils/ui/notify(options_stack).md

```diff
@@ -0,0 +1,138 @@
+---
+id: ui.notify(options,stack)
+module: ui/notify
+export: default
+---
+---
+##### shortDescription
+Creates a stackable toast message.
+
+##### param(options): Object
+The Toast [configuration](/api-reference/10%20UI%20Components/dxToast/1%20Configuration '/Documentation/ApiReference/UI_Components/dxToast/Configuration/').
+
+##### param(stack): Object
+An object with the following structure: *{position, direction}*.
+
+---
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos//WidgetsGallery/Demo/Toast/Stack/"
+}
+
+To stack toast messages, specify the `position` field in the **stack** object.
+
+The `position` field can be one of the following:
+
+- String    
+Available values:
+
+    - *'top left'*
+
+    - *'top center'*
+
+    - *'top right'*
+
+    - *'left center'*
+
+    - *'center'*
+
+    - *'right center'*
+
+    - *'bottom left'*
+
+    - *'bottom center'*
+
+    - *'bottom right'*
+
+- Object    
+Avaliable values: *{left | right, top | bottom}*.
+
+[note]
+
+If you use an object for `position` field, you can only specify one field in that object for vertical coordinate and one field for horizontal coordinate (for example, if you specify *'top'*, you do not need to specify *'bottom'*).
+
+[/note]
+
+You can also specify the `direction` field in the **stack** object. It is a string that can be one of the following:
+
+- *'up'*
+
+- *'down'*
+
+- *'left'*
+
+- *'right'*
+
+- *'up-reverse'*
+
+- *'down-reverse'*
+
+- *'left-reverse'*
+
+- *'right-reverse'*
+
+The image below shows the difference between directions with and without the 'reverse' postfix:
+
+![Toast Messages When Direction Is 'Up'](/images/Utilities/notify-stack-up.png) ![Toast Messages When Direction Is 'Up-Reverse'](/images/Utilities/notify-stack-up-reverse.png)
+
+When you set the `direction` to *'up'*, the second and subsequent toasts push the first toast upwards. When you set the `direction` to *'up-reverse'*, toasts stack on top of each other.
+
+If you do not specify the `direction` field, its default value is *'down'* for `position: 'top left | top center | top right'`, and *'up'* for other `position` options.
+
+Note that the default [hide](/Documentation/ApiReference/UI_Components/dxToast/Configuration/animation/#hide) animation duration for toast messages is 400 milliseconds. When you specify the `direction` with the 'reverse' postfix, set the **hide** animation duration to a lesser number or disable it to prevent the last toast in a stack from staying longer on the screen.
```

Comment:
Why didn't you put the same note in the other topic? I thought these were the same toast messages with the same settings. 

I have two comments for this text: 

- This sounds like a bug to me, especially when written like this. If I choose to stack notifications, then I assume that when the first one disappears, the rest shift closer to the edge. They don't remain in their positions. In other words, I would get stuck with a toast that was fourth in line and now hangs there alone in the middle of my screen. So... I don't mind that you want to mention the option, but I would use more neutral wording in hope that this behavior changes for the better in future releases. (That is, don't mention that there can be a single toast message staying in the middle of the screen for a while.) 
- "set to a lesser number". If you see such lengthy constructions, you can usually find a single verb to replace them. For example, I used "reduce". 

```suggestion
Note that toast messages disappear with an animated effect after their display time ends. The default [hide](/Documentation/ApiReference/UI_Components/dxToast/Configuration/animation/#hide) animation duration for toast messages is 400 milliseconds. If multiple messages can stack in your application and block important visual elements, you may want to disable the animation or reduce its duration.
```